src/SDKs/_metadata/datafactory_resource-manager.txt

```diff
@@ -1,11 +1,10 @@
-2017-12-07 02:33:12 UTC
+2017-12-08 05:40:35 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
```

Comment:
@smilenow I don't see the commit Id for the spec that is part of the meta data file?
Are you generating from the merged spec?

Reply:
@shahabhijeet Yes, I was generating from the Azure latest public repo yesterday, and didn't modify anything in this file. Do you mean it should input a certain commit id when using generate.cmd?

Reply:
@shahabhijeet , shall we go ahead with this PR? Or do you need a exact commit id when generating the SDK from the swagger spec?